sql/core/src/main/scala/org/apache/spark/sql/DataFrame.scala

```diff
@@ -1414,4 +1415,25 @@ class DataFrame private[sql](
     val jrdd = rdd.map(EvaluatePython.rowToArray(_, fieldTypes)).toJavaRDD()
     SerDeUtil.javaToPython(jrdd)
   }
+
+  /////////////////////////////////////////////////////////////////////////////
+  // Statistic functions
+  /////////////////////////////////////////////////////////////////////////////
+
+  // scalastyle:off
+  object stat {
```

Comment:
I think it looks like `df.stat$.MODULE$.freqItems()`. I don't know how we can otherwise make it `df.stat.freqItems` in scala.